apis/kueue/v1beta1/workload_types.go

```diff
@@ -59,6 +59,15 @@ type WorkloadSpec struct {
 	// +kubebuilder:default=""
 	// +kubebuilder:validation:Enum=kueue.x-k8s.io/workloadpriorityclass;scheduling.k8s.io/priorityclass;""
 	PriorityClassSource string `json:"priorityClassSource,omitempty"`
+
+	// QueueingPolicy that will determine if a job needs to be requeued or not after being suspended
```

Comment:
```suggestion
	// QueueingPolicy that will determine if a job can be queued.
```

Then you can expand about how it can be used to kick out a job that is running.